src/test/java/org/folio/fqm/repository/EntityTypeRepositoryTest.java

```diff
@@ -89,15 +89,15 @@ void shouldGetEntityTypeDefinitionWithCustomFields() {
         .visibleByDefault(true),
       new EntityTypeColumn()
         .name("custom_column_1")
-        .dataType(new BooleanType())
+        .dataType(new BooleanType().dataType("booleanType"))
```

Comment:
I'm not sure why these weren't here before, but it was causing test issues

Reply:
you mean earlier everything was happening magically lol :)